lib/config/__snapshots__/migration.spec.ts.snap

```diff
@@ -292,7 +292,6 @@ exports[`config/migration migrateConfig(config, parentConfig) migrates config 1`
   "separateMinorPatch": true,
   "suppressNotifications": [
     "lockFileErrors",
-    "deprecationWarningIssues",
```

Comment:
You need to test with a repo containing this option

Reply:
Tested again same link as above.

We could also add some migration logic to remove this value from the `suppressNotifications` but since it allows any string it won't be necessary.